packages/meditrak-app/ios/TupaiaMediTrak.xcodeproj/project.pbxproj

```diff
@@ -489,8 +491,13 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_ENTITLEMENTS = TupaiaMediTrak/TupaiaMediTrak.entitlements;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 140;
-				DEVELOPMENT_TEAM = 352QMCKRKJ;
+				DEVELOPMENT_TEAM = "";
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 352QMCKRKJ;
```

Comment:
Honestly, not that sure why Xcode felt the need to do this, but I trust Xcode more than I trust myself here